src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-form search-form-big" data-qa="big">
```

Comment:
According to task description please provide action and method for forms

- You have to put input inside the form for correct data processing. Form should have action and method attributes.